airflow/cli/commands/db_command.py

```diff
@@ -67,7 +67,7 @@ def shell(args):
                 host     = {url.host}
                 user     = {url.username}
                 password = {url.password or ""}
-                port     = {url.port or ""}
```

Comment:
I think it should depend on the database: Mysql = 3306, Postgres = 5432

Reply:
Thanks for your guidance, I should think more comprehensively